admin-api/authorization_server/internal_authorization.md

```diff
@@ -0,0 +1,95 @@
+As part of API extensibility, you can add new scopes to the API. This will allow you to create more granular permissions for your clients. You can find an [example module here](https://github.com/PrestaShop/example-modules/tree/master/api_module) and in [ps_apiresources](https://github.com/PrestaShop/ps_apiresources) repository.
+{{% /notice %}}
+
+After you save the client, you will see the client secret, but you won't be able to see it again. You can only see it once when you generate it. It's important to save it because you need it to authenticate your client.
+
+{{< figure src="../img/api_add_new_client3.jpeg" title="Remember to copy secret for your API Client" >}}
+
+## Getting an access token
+
+Now that you created your first API client, you can perform connection to the API. You can use Postman or any other tool to make HTTP requests.
+
+The first thing we are going to do is get the access token. You need to send a POST request to the `/admin-api/access_token` endpoint to do that. You need to provide the following parameters:
+
+* **client_id** - the ID of the client you created
+* **client_secret** - the secret of the client you created
+* **grant_type** - the type of grant you want to use. In this case, it's `client_credentials`
+* **scope** - the scopes you want to use. You can provide multiple scopes. In this case, we are going to use `api_client_read`, `api_client_write`, `customer_group_read`, `customer_group_write`.
+
+{{% notice %}}
+You have many tools to debug APIs. You can use Postman, [curl](https://curl.se/), or any other tool (like [Insomnia](https://insomnia.rest/)) that allows you to make HTTP requests. In this tutorial, I will use [Postman API client](https://www.postman.com/api-platform/api-client/), so that you can quickly test the API with a user-friendly interface.
+{{% /notice %}}
+
+Example request using curl:
+
+```bash
+curl --location 'http://yourdomain.test/admin-api/access_token' \
+--form 'grant_type="client_credentials"' \
+--form 'client_id="your_client_id"' \
+--form 'client_secret="your_client_secret"' \
+--form 'scope[]="api_client_read"' \
+--form 'scope[]="api_client_write"' \
+--form 'scope[]="customer_group_read"' \
+--form 'scope[]="customer_group_write"'
+```
+
+Example request using Postman:
+
+{{< figure src="../img/postman_access_token.png" title="Screenshot from Postman where we retrieve access token for our client" >}}
+
+{{% notice %}}
+As you can see in the screenshot, I am using project variables in Postman. This way, you can easily switch between different environments (like development, staging, and production) and don't have to change the request data (URL, client_id, etc.) every time you switch environments. You also won't find Authorization header in the request because it's generated automatically by Postman, thanks to ["Inherit authorization from parent"](https://learning.postman.com/docs/sending-requests/authorization/specifying-authorization-details/#inherit-authorization) option.
```

Comment:
```suggestion
As you can see in the screenshot, we are using project variables in Postman. This way, you can easily switch between different environments (like development, staging, and production) and don't have to change the request data (URL, client_id, etc.) every time you switch environments. You also won't find Authorization header in the request because it's generated automatically by Postman, thanks to ["Inherit authorization from parent"](https://learning.postman.com/docs/sending-requests/authorization/specifying-authorization-details/#inherit-authorization) option.
```